model-engine/model_engine_server/api/worker.py

```diff
@@ -1,8 +1,6 @@
 from uvicorn.workers import UvicornWorker
 
-# The target concurrency is around 50, so we set the limit to 32 with 4 workers
-# for a total concurrency of 128 to allow for some headroom.
-CONCURRENCY_LIMIT = 32
+CONCURRENCY_LIMIT = 1000
```

Comment:
I would add a comment here, basically mirroring the PR description. If I saw this I would be pretty confused without context.